style.css

```diff
@@ -0,0 +1,468 @@
+.body{

```

Comment:
oriunde dai inspect in pagina vine body-ul peste si nu poti selecta elementele, adauga un z-index foarte mic si un position relative, ca sa poti da inspect pe elemente